tools/ci_build/build.py

```diff
@@ -402,6 +402,12 @@ def convert_arg_line_to_args(self, arg_line):
 
     parser.add_argument("--ios", action="store_true", help="build for ios")
 
+    parser.add_argument(
+        "--macos",
+        choices=["MacOSX", "Catalyst"],
```

Comment:
```suggestion
        choices=["MacOSX", "Catalyst"],
        default="MacOSX",
```

Reply:
If the user must specify a value can the help text be updated to not have `Default is MacOSX`?

Reply:
right, that was out of date help text. removed.